### PR TITLE
chore(build): support arm64v8

### PR DESCRIPTION
**Pull request by qianlei90**
**What this PR does / why we need it**:

1. Makefile规范化
2. 支持arm64架构
3. 基础镜像从`cargo.caicloud.io/caicloud/debian:jessie`修改为`cargo.caicloud.xyz/library/debian:stretch`
4. 流水线规范化
5. golang版本升级到1.13.6

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to http://jira.caicloud.xyz/browse/CPS-5949

**Special notes for your reviewer**:

/cc @hezhizhen @whalecold @bbbmj @liwandaniel @supereagle 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,24 +1,143 @@
...
+
+build-linux:
+	@docker run --rm -t                                                                \
+	  -v $(PWD):/go/src/$(ROOT)                                                        \
+	  -w /go/src/$(ROOT)                                                               \
+	  -e GOOS=linux                                                                    \
+	  -e GOARCH=$(ARCH)                                                                \
+	  -e GOPATH=/go                                                                    \
+	  -e SHELLOPTS=$(SHELLOPTS)                                                        \
+	  $(BASE_REGISTRY)/golang:1.13.6-stretch                                           \
```
Comment (whalecold) [inline Makefile]: 应该用 1.10.4 版本的 golang

Comment (qianlei90) [reply Makefile]: done，流水线的要改成1.10.4吗？

Comment (qianlei90) [reply Makefile]: 流水线上的`make test`和`make lint`我也改成1.10.4了

**Pull request merged**